doc-content/jbpm-docs/src/main/asciidoc/Persistence/transaction-retry-con.adoc

```diff
@@ -0,0 +1,33 @@
+[id='transaction-retry-con_{context}']
+= Transaction retries
+
+When the {PROCESS_ENGINE} commits a transaction, sometimes the committing fails because another transaction is being committed at the same time. In this case, the {PROCESS_ENGINE} needs to retry the transaction.
```

Comment:
Minor edit suggestion:

When the {PROCESS_ENGINE} commits a transaction, sometimes the commit fails because another transaction is being committed at the same time. In this case, the {PROCESS_ENGINE} must retry the transaction.